src/docs/Components/Pagination/index.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: 'Pagination'
+order: 8
+description: "The pagination enable users to navigate from a page to another one, especially when you have to split a large content in several parts. It's mainly used on pages like the search engine result page or the list pages."
```

Comment:
Pagination enables users to navigate from a page to another one, especially when a large content needs to be splitted in several parts. It's mainly used on pages like the search engine result page or the list pages.